### ISSUE TITLE
Link color has a poor contrast

### ISSUE THREAD

**Issue by not-my-profile** [ruma/ruma.github.io#24]
![image](https://user-images.githubusercontent.com/73739153/143625752-9e1e809b-5651-4dd0-a8b3-76f437db72d2.png)

WCAG recommends a contrast of at least 4.5. Inspecting the color in Chromium/Firefox developer tools also shows that this contrast doesn't meet WCAG.

**Comment by jplatte** [comment]
I guess we need to use a darker shade then. Want to open a PR?

**Comment by not-my-profile** [comment]
Sorry I don't have the time. I'd advise switching to some dark blue color to make it more obvious that those are links.

**Comment by jplatte** [comment]
Fixed in 929741a2a18b32fa2b2233e2b4038cb68bf36713. I didn't find a shade of yellow with okay contrast so had to go with blue.

**Comment by not-my-profile** [comment]
Thank you!
